packages/libs/client/etc/client.api.md

```diff
@@ -24,6 +24,12 @@ export function buildUnsignedTransaction(parts: string[], holes: string[], args:
 // @alpha (undocumented)
 export function createPactCommandFromTemplate(tpl: IPactCommand): PactCommand;
 
+// @alpha (undocumented)
+export interface ICapV2 {
+    // (undocumented)
+    addCap(cap: string, signer: string, ...args: PactValue[]): this;

```

Comment:
I would love to have a more verbose API like `addCapabilities` as kt is more clear and more inline with our renewed domain model